js/north.js

```diff
@@ -64,29 +64,33 @@ jQuery( function ( $ ) {
 
 	// Remove the no-touch body class for touch devices
 	var isTouchDevice = 'ontouchstart' in document.documentElement;
-	if( isTouchDevice ) {
+	var msTouchEnabled = window.navigator.msMaxTouchPoints;
```

Comment:
Is this second variable needed? Come to think of it, is there a good reason why isTouchDevice is needed? If it's purely for descriptive purposes, then you could just add in comments and put everything in the if statement instead.

```
// This this is a touch device. We detect this through xxxxxx and xxxxx.
if( 'ontouchstart' in document.documentElement || window.navigator.msMaxTouchPoints )
...
```